va/va.h

```diff
@@ -2350,6 +2351,8 @@ typedef struct _VAPictureParameterBufferH264
         unsigned int value;
     } pic_fields;
     unsigned short frame_num;
+    unsigned char num_ref_idx_l0_default_active_minus1;
+    unsigned char num_ref_idx_l1_default_active_minus1;
```

Comment:
it is from h264 spec. it will take effect when num_ref_idx_active_override_flag in slice header is false